.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,31 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: bug
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+
+Steps to reproduce the behavior.
```

Comment:
````suggestion
**Describe how to cause this behavior**

What did you do to get it to happen?
Does it happen every time you follow these steps, sometimes, or only one time?

```
If you have a traceback, please paste it here; otherwise, delete this.
```

````